docs/development.md

```diff
@@ -278,3 +278,15 @@ I0511 10:15:29.585389 244858 hello_world.cc:50] Request: WORLDS
 I0511 10:15:29.591434 244858 hello_world.cc:57] Response: HELLO WORLDS!
 I0511 10:15:29.593106 244858 hello_world.cc:57] Response: HELLO AGAIN WORLDS!
 ```
+
+### Signing Oak Wasm modules
+
+Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
```

Comment:
Is it explained anywhere how these signatures are used (for example for determining the privileges, if I understand correctly)? 

There was also a comment from Tiziano about modules with multiple signatures. Is it possible to make multiple signatures with this command or would that require a different command? 

There is some interesting discussion and description of what these signatures are for in #1274. I think it would be nice to have a summary documented somewhere in our docs, if it is not already there.